src/CustomerioTracking.tsx

```diff
@@ -35,7 +36,12 @@ class CustomerIO {
    * @returns 
    */
    static initialize(siteId: string, apiKey: string, region: Region = Region.US, config: CustomerioConfig = new CustomerioConfig()) {
-    return CustomerioReactnative.initialize(siteId, apiKey, region, config)
+
+    let pversion = pjson.version
```

Comment:
You could do `let pversion = pjson.version ?? ''` then the `if` statement below won't be needed

Reply:
Make sense. Thanks Segun !